pkg/kubectl/cmd/get_test.go

```diff
@@ -217,22 +217,22 @@ func TestGetObjectsWithOpenAPIOutputFormatPresent(t *testing.T) {
 }
 
 type FakeResources struct {
-	resources map[schema.GroupVersionKind]openapi.Schema
+	resources map[string]openapi.Schema
 }
 
-func (f FakeResources) LookupResource(s schema.GroupVersionKind) openapi.Schema {
+func (f FakeResources) LookupResource(s string) openapi.Schema {
```

Comment:
I'm not super excited about this interface change. We're going from a well structured argument to a arbitrarily selected formatting of multiple strings. I would have loved a `group, version, kind string` list of argument much more.
And changed `ParseGroupVersionKind` to return `group, version, kind string` so that it could still be fed directly as a parameter to that interface.